sources/nodes/kube.go

```diff
@@ -78,8 +78,15 @@ func (self *kubeNodes) List() (*NodeList, error) {
 			nodeList.Items[Host(node.Name)] = Info{PublicIP: node.Status.HostIP, InternalIP: addrs[0].String()}
 			goodNodes = append(goodNodes, node.Name)
 		} else {
-			glog.Errorf("Skipping host %s since looking up its IP failed - %s", node.Name, err)
-			self.recordNodeError(node.Name)
+			ip := net.ParseIP(node.Name)
```

Comment:
Yes, the node names(not hostname) match the IP of the node because on RackSpace you can't resolve the name of other machines, if you want to do this, you must declare each machines in your hosts file or add a local dns mechanism.